### PR TITLE
Ids 488 improve the mark ingested step for the confusion

### PR DESCRIPTION
**Pull request by LibbyLi667**
Adjusted YAML helper to better represent "data_status", "data_classification", "Path" in YAML file and can be accepted by IDW.
Removed creating a list of newly ingested datafiles for the conveyor.

### CODE REVIEW

File: src/profiles/idw/yaml_helper.py [hunk 1]
```diff
@@ -141,21 +142,10 @@ def __init__(
         yaml.constructor.SafeConstructor.add_constructor(
             path_tag, self._path_constructor
         )
-        yaml.add_representer(Path, self._path_reprer)
+        yaml.add_representer(PosixPath, self._path_reprer)
```
Comment (andrew-uoa) [inline src/profiles/idw/yaml_helper.py:145]: Just wondering why we need to use `PosixPath` specifically here? `PosixPath` cannot be instantiated on Windows, and while Windows isn't our primary OS, it's nice to keep that portability if possible.

**Review by andrew-uoa — COMMENTED** [review]
Looks good on the whole, I just left one question

File: src/profiles/idw/yaml_helper.py (hunk 1 above)
Comment (LibbyLi667) [reply src/profiles/idw/yaml_helper.py:145]: good point! But the ingestion script will be running on a server which is a Linux system, and this wouldn't affect the YAML file, which might be on a Windows machine, so should be fine? PS: could test on a Windows machine if needed.

**Review by andrew-uoa — APPROVED** [review]
Looks good to me 😄

**Pull request merged**